packages/gas-fee-controller/src/fetchBlockFeeHistory.test.ts

```diff
@@ -71,20 +73,20 @@ describe('fetchBlockFeeHistory', () => {
 
       expect(feeHistory).toStrictEqual([
         {
-          number: new BN(1),
-          baseFeePerGas: new BN(10_000_000_000),
+          number: fromHex(toHex(1)),
```

Comment:
I think we might have brought in a slightly different version of `bn.js` with these changes which is causing these tests to fail now. This way is closer to how these values are represented as they are passed to `fetchBlockFeeHistory` above, anyway.

Reply:
The version difference causing these failures seems to have been eliminated. Though, it seems OK to leave this as is